axonserver/src/main/java/io/axoniq/axonserver/grpc/PlatformService.java

```diff
@@ -180,13 +180,13 @@ protected void consume(PlatformInboundInstruction instruction) {
                                                                     topology.getMe().getName(),
                                                                     sendingStreamObserver);
                 } else {
-                    handlers.getOrDefault(requestCase, new ArrayDeque<>())
-                            .forEach(consumer -> {
-                                instructionAckSource.sendSuccessfulAck(instruction.getInstructionId(),
-                                                                       sendingStreamObserver);
-                                consumer.accept(this.clientComponent.get(),
-                                                instruction);
-                            });
+                    instructionAckSource.sendSuccessfulAck(instruction.getInstructionId(),
```

Comment:
If we don't have a handler for the request case I think we should NOT send the ack, right?